CMake/cdat_modules/rpy2_external.cmake

```diff
@@ -0,0 +1,11 @@
+# create an external project to install RPY2,
+# and configure and build it
+set(nm RPY2)
+
+# Set PATH and R_HOME to find R
+list(APPEND USR_ENVS
+  "R_HOME=${cdat_EXTERNALS}/lib/R"
```

Comment:
@doutriaux1 CDAT_BUILD_R will be ON if rpy2 is building since we are not supporting system R.